content/en/blog/recap-2022/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Falco: Project Recap 2022"
+slug: recap-2022
+date: 2022-12-20
+author: Jacqueline Salinas
+images:
+  - /blog/recap-2022/images/recap-2022-featured.png
+---
+
+![](images/recap-2022-featured.png)
+
+Dear Falco community, 
+
+As we write this, it’s unbelievable that 2022 is wrapping up and very soon! As always, we are truly grateful for the amazing community of maintainers and contributors. The Falco project and community would not be the vibrant ecosystem it is today without these incredible individuals. Let’s take a look at what the community accomplished in 2022, and we’ll give a quick glimpse of what to expect in 2023!  
```

Comment:
We should update this section, as we're in 2023 now

Reply:
```suggestion
As always, we are truly grateful for the amazing community of maintainers and contributors. The Falco project and community would not be the vibrant ecosystem it is today without these incredible individuals. Let’s take a look at what the community accomplished in 2022, and we’ll give a quick glimpse of what to expect in 2023!  
```